diff --git a/tests/ffip-tests.el b/tests/ffip-tests.el
index a54fd6d..0c4738d 100644
--- a/tests/ffip-tests.el
+++ b/tests/ffip-tests.el
@@ -37,6 +37,20 @@
     (setq files (mapcar 'car (ffip-project-search nil nil)))
     (should (> (length files) 1))
     (should (not (active-minibuffer-window)))
+
+    (setq ivy-read-called nil)
+    ;; ffip will call ivy by default
+    (ffip-find-files nil nil)
+    (should ivy-read-called)))
+
+(ert-deftest ffip-ffip-show-diff ()
+  (let (files
+        (ffip-diff-backends '((let (str "")
+                                str))))
+    (setq ffip-project-root default-directory)
+    (setq files (mapcar 'car (ffip-project-search nil nil)))
+    (should (> (length files) 1))
+    (should (not (active-minibuffer-window)))
     (setq ivy-read-called nil)
     ;; ffip will call ivy by default
     (ffip-find-files nil nil)
diff --git a/tests/git-diff.diff b/tests/git-diff.diff
index bacd2be..3a4d83d 100644
--- a/tests/git-diff.diff
+++ b/tests/git-diff.diff
@@ -9,6 +9,6 @@
                            history preselect keymap update-fn sort
                            action unwind re-builder matcher dynamic-collection caller)
   (setq ivy-read-called t)
-  (message "ivy-read mockup is called"))
+  (message "`ivy-read' mockup is called"))
 (provide 'ivy)
\ No newline at end of file
